_posts/2018-05-29-Rust-1.26.1.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Announcing Rust 1.26.1"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.26.1. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.26.1 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.26.1][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1261-2018-05-29
```

Comment:
Traditionally this link points to master, not stable. I don't think it's a *problem* exactly...

Reply:
The release notes are not merged into master yet though.

Reply:
Hopefully they will be by tuesday? Usually they're not ready when I write the notes :)